src/components/GlobalStyle.tsx

```diff
@@ -1,28 +1,40 @@
 import {
-  black,
-  green,
+  blueGrey300,
```

Comment:
I spoke to the designers and we've come up with this naming scheme that standardizes these colors across all our apps.